x/ibc/03-connection/handler.go

```diff
@@ -0,0 +1,126 @@
+package ics03
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/ibc/03-connection/keeper"
+	"github.com/cosmos/cosmos-sdk/x/ibc/03-connection/types"
+)
+
+// NewHandler creates a new Handler instance for IBC connection
+// transactions
+func NewHandler(k keeper.Keeper) sdk.Handler {
```

Comment:
Again I would somewhat prefer these were all exposed in a single outward-facing module (a la ICS 25).

Up to the SDK team though.

Reply:
Oh yeah, I agree. I've only reached ICS05... there's a long way to go til ICS25 lol

Reply:
Luckily there are no ICSs from 6 to 19.